### DISCUSSION TITLE
Habe die Firmensoftware überschrieben

### DISCUSSION THREAD

**Discussion by Quadrunner3010** [community/community#131246] [Actions]
### Select Topic Area

Question

### Body

Hi, ich habe mir ein Deauther Monster V5 gekauft. Bei der Anmeldung ist irgendetwas schief gelaufen. Desshalb funktioniert der Monitor nicht mehr. Wie kann ich den Deauther zurücksetzen, so wie ich den bekommen habe.
Danke

**Comment by ossanna16** [comment]
Hi @Quadrunner3010 , thanks for participating!

👋 and thank you for being a part of our community. To ensure a safe and supportive environment, we look to our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) which states all communications here on the Community needs to be in English. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
